.github/workflows/docker.yml

```diff
@@ -0,0 +1,51 @@
+name: Docker
+
+on:
+  push:
+    tags:
+    branches:
+      - main
```

Comment:
I believe the main branch name for Rosette is still `master`; changing it to be `main` now will break dependencies that assume branch names. If we want to change this we should consider it separately from this PR to understand what all will break.

Reply:
Good point! It was not me trying to push for `main`, just me copying a bit too fast from my other repositories. Fixed in a9c6bf6.